lib/core/test/unit/Cardano/Wallet/Api/Malformed.hs

```diff
@@ -992,6 +993,17 @@ instance Malformed (BodyParam ApiPostRandomAddressData) where
           )
         ]
 
+instance Malformed (BodyParam SettingsPutData) where
+    malformed = first (BodyParam . Aeson.encode) <$>
+        [ ( [aesonQQ|
+            { "settings": {
+                          "pool_metadata_source": "not_a_uri"
+                          }
+            }|]
+          , "Error in $['address_index']: parsing Int failed, expected Number, but encountered String"
```

Comment:
That error message does not sound right **at all**.

Reply:
Copy pasta lol